spk/jellyfin/src/service-setup.sh

```diff
@@ -10,7 +10,11 @@ JELLYFIN_ARGS="--service \
  -w ${SYNOPKG_PKGDEST}/web \
  --ffmpeg /var/packages/ffmpeg/target/bin/ffmpeg"
 
-SERVICE_COMMAND="env LD_LIBRARY_PATH=${SYNOPKG_PKGDEST}/lib ${SYNOPKG_PKGDEST}/share/jellyfin ${JELLYFIN_ARGS}"
+if [ ${SYNOPKG_DSM_VERSION_MAJOR} -lt 7 ]; then
+    SERVICE_COMMAND="env LD_LIBRARY_PATH=${SYNOPKG_PKGDEST}/lib ${SYNOPKG_PKGDEST}/share/jellyfin ${JELLYFIN_ARGS}"
+else
+    SERVICE_COMMAND="env ${SYNOPKG_PKGDEST}/share/jellyfin ${JELLYFIN_ARGS}"
+fi
```

Comment:
When patching the rpath there is not need to set LD_LIBRARY_PATH
```suggestion
SERVICE_COMMAND="${SYNOPKG_PKGDEST}/share/jellyfin ${JELLYFIN_ARGS}"
```

Reply:
Hmm, much simpler. Would this be the preferred way to approach the ***arr** apps as well?

Reply:
> Hmm, much simpler. Would this be the preferred way to approach the ***arr** apps as well?

No, the *arr packages use the internal package updater and updated executables are out of our control. 
IMHO the LD_LIBRARY_PATH is the only solution for such packages.

The use of patchelf was the only solution for dotnet-runtime as there is no service configuration to place a LD_LIBRARY_PATH.
In jellyfin we have a service starter so we can use either solution (to make the service-setup independent of DSM Version I prefer the patchelf here).